rpqueue/__init__.py

```diff
@@ -740,6 +743,8 @@ def function2(arg1, arg2, ...):
             'will execute from within the 'default' queue.'
     '''
     queue = kwargs.pop('queue', None) or b'default'
+    if PY3 and isinstance(queue, str):
+        queue = queue.encode()
```

Comment:
This is fine but just fyi -- checking `PY3` is probably not necessary (`.encode()` works on str just fine in py2)

Reply:
We don't need to encode in python 2. It is already encoded